### PR TITLE
hw05

### PR DESCRIPTION
**Pull request by LenaK513**

### CODE REVIEW

**Comment by LenaK513** [comment]
HW without test part

**Comment by Harryk13** [comment]
Do you want to finish the test part before?

File: controllers/users/updateAvatar.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+const path = require("path");
+const fs = require("fs/promises");
+
+const avatarDir = path.join(__dirname, "../../", "public", "avatars");
+
+const updateAvatar = async (req, res, next) => {
+  const { path: tempUpload, originalname } = req.file;
+  const { _id: id } = req.user;
+  const imageName = `${id}_${originalname}`;
```
Comment (Harryk13) [inline controllers/users/updateAvatar.js:13]: What happens if a user downloads the same file multiple times?

Comment (LenaK513) [reply controllers/users/updateAvatar.js:13]: nothing happens

Comment (Harryk13) [reply controllers/users/updateAvatar.js:13]: It's incorrect. It may be another file with same name and you must rewrite this file.

Comment (LenaK513) [reply controllers/users/updateAvatar.js:13]: I am not going to make any improvements or changes in this homework

**Pull request merged**